tests/e2e/examples/render.sh

```diff
@@ -54,58 +77,100 @@ render_install_example "$example_name" "01"
+if [[ $IS_OPENSHIFT = true && $SKIP_ES_EXTERNAL = true ]]; then
+    $YQ e -i '.spec.storage.options={}' ./01-install.yaml
+    $YQ e -i '.spec.storage.elasticsearch={"nodeCount":1,"resources":{"limits":{"memory":"2Gi"}}}' ./01-install.yaml
+else
+    render_install_elasticsearch "upstream" "00"
+fi
 render_smoke_test_example "$example_name" "02"
 
 
+###############################################################################
+# TEST NAME: examples-simple-prod-with-volumes
+###############################################################################
 start_test "examples-simple-prod-with-volumes"
 example_name="simple-prod-with-volumes"
-render_install_elasticsearch "upstream" "00"
 render_install_example "$example_name" "01"
+if [[ $IS_OPENSHIFT = true && $SKIP_ES_EXTERNAL = true ]]; then
+    $YQ e -i '.spec.storage.options={}' ./01-install.yaml
+    $YQ e -i '.spec.storage.elasticsearch={"nodeCount":1,"resources":{"limits":{"memory":"2Gi"}}}' ./01-install.yaml
+else
+    render_install_elasticsearch "upstream" "00"
+fi
 render_smoke_test_example "$example_name" "02"
 $GOMPLATE -f ./03-check-volume.yaml.template -o 03-check-volume.yaml
 
 
+###############################################################################
+# TEST NAME: examples-simplest
+###############################################################################
 start_test "examples-simplest"
 example_name="simplest"
 render_install_example "$example_name" "00"
 render_smoke_test_example "$example_name" "01"
 
 
+###############################################################################
+# TEST NAME: examples-with-badger
+###############################################################################
 start_test "examples-with-badger"
 example_name="with-badger"
 render_install_example "$example_name" "00"
 render_smoke_test_example "$example_name" "01"
 
 
+###############################################################################
+# TEST NAME: examples-with-badger-and-volume
+###############################################################################
 start_test "examples-with-badger-and-volume"
 example_name="with-badger-and-volume"
 render_install_example "$example_name" "00"
 render_smoke_test_example "$example_name" "01"
 
 
+###############################################################################
+# TEST NAME: examples-with-cassandra
+###############################################################################
 start_test "examples-with-cassandra"
 example_name="with-cassandra"
 render_install_cassandra "00"
 render_install_example "$example_name" "01"
+# jaeger-cassandra-schema multi arch image is not available
+# created a PR: https://github.com/jaegertracing/jaeger/pull/4122 and https://github.com/jaegertracing/jaeger/pull/4123
+# until we get new release image, this is a workaround image
+# $YQ e -i '.spec.storage.cassandraCreateSchema.image = "quay.io/redhat-distributed-tracing-qe/jaeger-cassandra-schema:1.40.0"' ./01-install.yaml
```

Comment:
I don't get it. Do we need running this command or not?

Reply:
@iblancasa sorry, we have to workaround. update the image on the operator. hence it is not required here. I will remove those lines